docker/app
# Fix 'env_file' retrieval by using app.Path as WorkingDir

Fixes #583

**- What I did**
Fix `env_file` retrieval

**- How I did it**
Pass app.Path as WorkingDir for retrieving the `env_file`

**- How to verify it**
On a `docker app` project that includes an environment file (in the same folder as the docker-compose.yml) run:
```
$ docker app render
```
The render should be run with no errors

**- Description for the changelog**
Fixed environment file (`env_file`) retrieval when rendering

**- A picture of a cute animal (not mandatory but encouraged)**
![white](https://user-images.githubusercontent.com/373485/63452328-4a035d00-c446-11e9-914b-e64e29e318af.jpg)
